source/coding-the-legislation/writing_yaml_tests.md

```diff
@@ -80,54 +80,54 @@ In this case, there is another convention:
 - do not include the field `input_variables` but instead define new keys corresponding to the entities:
 
     ```yaml
-    - name: "IRPP - Famille ayant des revenus salariaux de 20 000 €"
+    - name: "IRPP - Family with salary income of 20,000 €"
     period: 2012
     absolute_error_margin: 0.5
     input:
-      familles:
-      menages:
-      foyers_fiscaux:
+      families:
+      households:
+      tax_homes:
```

Comment:
`fiscal_unit` ?

Reply:
i don't know what a "fiscal_unit" is.

Reply:
could it be "taxable units"

Reply:
You should be right, I googled and I found "tax_household".
I think the term is so linked to a precise country that it is more confusing to try to translate because you add a layer of potential confusion. So I favor using the french terms for the french tax and benefit system